src/handwritten/task.py

```diff
@@ -402,6 +402,28 @@ def is_task_done(self):
 
         return is_task_done
 
+    def perform_bridge_offset_nulling_cal(self, channel="", skip_unsupported_channels=False):
+        """
+        Performs a bridge offset nulling calibration on the channels in the task.
+        If the task measures both bridge-based sensors and non-bridge-based sensors,
+        use the channels input to specify the names of the channels that measure 
+        bridge-based sensors.
```

Comment:
The first line of the docstring is a one-line summary and should not be part of a larger paragraph.

The summary line should be a command ("Perform" not "Performs") as per https://peps.python.org/pep-0257/#one-line-docstrings

Reply:
Thanks for pointing this out, I have separated it into one-line summary and the rest of the explanation is one empty line below.